share/qtvcp/screens/qtplasmac/qtplasmac_handler.py

```diff
@@ -4801,7 +4801,7 @@ def delete_material_clicked(self):
             except:
                 if not matNum:
                     msg0 = _translate('HandlerClass', 'A material number is required')
-                    msgs = f'{msg0}.\n\n\{msg1}:'
+                    msgs = f'{msg0}.\n\n\\{msg1}:'
```

Comment:
```suggestion
                    msgs = f'{msg0}.\n\n{msg1}:'
```
Could this also have been the author's intention? I have no idea how these commands should look like.

Reply:
>Could this also have been the author's intention?

I would prefer not to guess rather wait for the author to comment.

Reply:
> Could this also have been the author's intention? I have no idea how these commands should look like.

Yes, this was a fairly long standing typo, thanks for noticing; hansu's version is correct.